shared/version/api.go

```diff
@@ -388,7 +388,6 @@ var APIExtensions = []string{
 	"instance_move_config",
 	"ovn_ssl_config",
 	"init_preseed_storage_volumes",
-	"metrics_instances_count",
```

Comment:
@gabrielmougard @simondeziel but arent we keeping the metrics instances count part?

Reply:
Right this should stay. Now the extension described it added instances count but didn't mentioned anything about offline instance metrics. I'm not sure how to gracefully handle this situation.

Maybe we can assume the population depending on `metrics_instances_count` to also come with offline instance metrics is so small that we simply proceed and call it out in the release announcement?

Reply:
yeah i think sp

Reply:
fixed